### ISSUE TITLE
`Dockerfile` in repo?

### ISSUE THREAD

**Issue by jakirkham** [conda-forge/ldcpy-feedstock#23]
Is there a reason this `Dockerfile` is here?

https://github.com/conda-forge/ldcpy-feedstock/blob/99f4860935cefa0e970ecfab19ca51e95a830ed6/.ci_support/Dockerfile#L1-L2

**Comment by jakirkham** [comment]
cc @conda-forge/core

**Comment by beckermr** [comment]
Whoa. Does that extras extend base key even work? I've never seen this before in my life.

**Comment by jakirkham** [comment]
Yeah actually not sure how `apt-get` would work here since the base image is using CentOS

**Comment by jakirkham** [comment]
Appears this was introduced in PR ( https://github.com/conda-forge/ldcpy-feedstock/pull/2 ). Particularly commit ( https://github.com/conda-forge/ldcpy-feedstock/commit/6ec7e7527e09f77999bffc0d57e5a511d59d582c )

@pinarda, could you please explain what this file is doing here?
